heareval/tasks/util/luigi.py

```diff
@@ -107,23 +107,21 @@ def stage_number(self) -> int:
 class ExtractArchive(WorkTask):
     
     infile = luigi.Parameter()
+    #The previous task will be passed in as a parameter.
+    #This must have workdir attribute and the zip file to extract should be stored inside it.
+    prev_task = luigi.TaskParameter()
```

Comment:
Added a `luigi.TaskParameter` as input to take the previous task. This might solve the problem of reusing the tasks from the utils.

Reply:
I really don't think we should do this. We can just return DownloadCorpus(url) in self.requires()

Reply:
Changed in the new commit.